free-programming-books-pl.md

```diff
@@ -72,6 +82,13 @@
 * [zeimer.github.io (Programowanie Funkcyjne)](https://zeimer.github.io)
 
 
+### CSharp
+
+* [4programmers tutorial](http://4programmers.net/C_sharp) - kurs z lukami, zawiera łatwe do zrozumienia treści.
+* [Cezary Walenciuk](http://cezarywalenciuk.pl/blog/programing/category/c%23) - Blog i kurs C# w jednym.
```

Comment:
This is a blog. Do you think it is important enough, or book-like enough, to overcome our hesitance to list blogs?

Reply:
afther review i think so that 4programmers is scruffy, backyard style. for consistent quality it would be better remove. Second link seems to be fine.